mlir/test/Conversion/AMDGPUToROCDL/amdgpu-to-rocdl.mlir

```diff
@@ -9,6 +9,7 @@
 // test pass doesn't set up the GPU address space conversions.
 
 #gpu_global_addrspace = 1
+#gpu_lds_addrspace = 3
```

Comment:
This is an ugly hack for not having the address space conversions set up in `-amdgpu-to-rocdl`. Is fine as is, but wanted to flag it